editors/vscode/src/common.ts

```diff
@@ -165,7 +170,7 @@ export function activate(
                 "workspace/didChangeConfiguration",
                 { settings: "" },
             );
-            wasm_preview.refreshPreview();
+            // wasm_preview.refreshPreview();
```

Comment:
I guess now the lsp will tell the preview to refresh and that's why it's no longer needed.
But the line can go away